java-iam/.github/trusted-contribution.yml

```diff
@@ -1,3 +0,0 @@
-trustedContributors:
```

Comment:
We need to keep this file in the root if we still want to rely on owlbot to update `java-iam` and `java-common-protos`